docs/pages/access-controls/sso/google-workspace.mdx

```diff
@@ -4,228 +4,279 @@ description: How to configure Teleport access with Google Workspace (formerly kn
-a member of one group by virtue of being in another group. This happens when a child
-group is nested within a parent group, so that a member of a child group is also
+The key property that Teleport relies on to support Google Workspace is **transitive group membership**.
+With transitive group membership, a user can be a member of one group by virtue of being in another group. 
+For example, if there is a child group nested within a parent group, every member of the child group is also
 a member of the parent group.
 
+#### Google Workspace editions and API
+
 Google Workspace service accounts can determine whether a user has a transitive
-membership to a specific group by calling methods within the Google Workspace
+membership in a specific group by calling methods in the Google Workspace
 **Cloud Identity API**. These API methods are only usable for users belonging to
-specific Google Workspace plans:
-
-- Enterprise Standard
-- Enterprise Plus
-- Enterprise for Education
-- Cloud Identity Premium
+specific Google Workspace editions.
 
 The Google Workspace **Directory API** allows administrators to list users and
 groups in their Google Workspace domain, but does not enable them to query
 transitive group memberships. The Directory API is available for all Google
-Workspace plans.
+Workspace editions.
 
 #### How Teleport uses Google Workspace APIs
 
-Teleport's OIDC connector uses Google Workspace's APIs differently depending
-on the resource version.
+The Teleport OIDC connector uses the Google Workspace APIs to map users in Teleport 
+roles to the Google Workspace groups they belong to.
+
+To list a user's Google Workspace groups, Teleport first attempts to
+get credentials by calling Cloud Identity API methods. If successful, Teleport uses 
+the credentials to query a user's transitive group memberships.
+
+If the credentials don't exist, Teleport gets credentials by calling 
+Directory API methods. Teleport then uses the Directory API to list the 
+user's groups in the entire Google Workspace for your organization. 
+Groups that the user belongs to that are external to the workspace aren't listed.
+
+#### How to check your current edition
+
+To determine whether your Google Workspace edition supports querying transitive
+group memberships:
+
+1. Open [Inspect Groups](https://admin.google.com/ac/groupsinspection)
+in the Google Workspace Admin Console. 
+   
+   Group inspection relies on the Cloud Identity API. 
+   
+1. Select **List all groups for a member** and **Include external groups**.
+   
+   If your Google Workspace edition supports the Cloud Identity API, you should
+   block access to external groups at the workspace level as described in [Allow or block
+   access to external groups](https://support.google.com/a/answer/9468710). Otherwise, membership 
+   in any group external to the workspace will prevent users from signing in.
+   
+   If your Google Workspace edition doesn't support the Cloud Identity API, you must
```

Comment:
Removes superfluous indents on newlines.

```suggestion

To determine whether your Google Workspace edition supports querying transitive
group memberships:

1. Open [Inspect Groups](https://admin.google.com/ac/groupsinspection)
in the Google Workspace Admin Console. 

   Group inspection relies on the Cloud Identity API. 

1. Select **List all groups for a member** and **Include external groups**.

   If your Google Workspace edition supports the Cloud Identity API, you should
   block access to external groups at the workspace level as described in [Allow or block
   access to external groups](https://support.google.com/a/answer/9468710). Otherwise, membership 
   in any group external to the workspace will prevent users from signing in.

   If your Google Workspace edition doesn't support the Cloud Identity API, you must
```